include/seqan3/alignment/matrix/alignment_coordinate.hpp

```diff
@@ -7,320 +7,20 @@
 
 /*!\file
  * \author Marcel Ehrhardt <marcel.ehrhardt AT fu-berlin.de>
- * \brief Provides seqan3::detail::alignment_coordinate.
+ * \brief Provides seqan3::alignment_coordinate.
  */
```

Comment:
The file itself should be deprecated and there should be a
```
SEQAN3_DEPRECATED_HEADER(
  "This header is deprecated and will be removed along all its content in SeqAn-3.1.0.")
```

Or will something survive in this header?

Reply:
No it won't. Do you know how the tests will handle it if I include a deprecated header?

Normally we would just include the new header, but in this case there is no new header. So I opted to not deprecate the header.

I could move the deprecated alignment_coordinate again into the other header and include that...